src/kata4.humanCatDogYears.js

```diff
@@ -1,3 +1,20 @@
-const humanCatDogYears = (number) => {};
+const humanCatDogYears = (number) => {
+  const humanAge = number;
+  const catYear = number + 14;
+  const dogYear = number + 14;
+
+
+  if (humanAge === 0) {
+    return [0, 0, 0];
+  }
+
+  if (humanAge === 1) {
+    return [humanAge, catYear, dogYear];
+  }
+
+  if (humanAge > 1) {
+    return [humanAge, humanAge * 4 + 16, humanAge * 5 + 14];
+  }
```

Comment:
This does work, but it can be improved a bit. For one thing, you neednt create the `humanAge` variable, you could just use the `number` variable being passed in directly. Also, could remove 1 of the 2 blank lines between the variable declarations and the if statements:
```
  const catYear = number + 14;
  const dogYear = number + 14;

  if (number === 0) {
    return [0, 0, 0];
  }

  if (number === 1) {
    return [number, catYear, dogYear];
  }

  if (number > 1) {
    return [number, number * 4 + 16, number * 5 + 14];
  }
```